tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -138,6 +138,7 @@ const fileIconWrapper = '#oc-file-details-sidebar .details-icon-wrapper'
 const fileIconPreview = '#oc-file-details-sidebar .details-preview'
 const activitySidebarPanel = 'sidebar-panel-activities'
 const activitySidebarPanelBodyContent = '#sidebar-panel-activities .sidebar-panel__body-content'
+const contextmenuAction = '//*[@id="oc-files-context-actions-context"]//span[text()="%s"]'
```

Comment:
```suggestion
const contextMenuAction = '//*[@id="oc-files-context-actions-context"]//span[text()="%s"]'
```
you can use xpath axes if needed and better choice